core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/util/LinkInspectorResourceUtil.java

```diff
@@ -59,6 +52,36 @@ public static void removeResource(String path, ResourceResolver resourceResolver
         }
     }
 
+    public static void createNode(String path, ResourceResolver resourceResolver) {
+        try {
+            Session session = resourceResolver.adaptTo(Session.class);
+            if (session == null) {
+                LOG.warn("Session is null, recreating node is interrupted");
+                return;
+            }
+            JcrUtil.createPath(path, JcrConstants.NT_UNSTRUCTURED, session);
+            session.save();
+        } catch (RepositoryException e) {
+            LOG.error(String.format("Failed to recreate node %s", path), e);
```

Comment:
Until Java 17, String formatter operations are quite inefficient. Please avoid them whenever you can. In the current case, Message Formatter works better:
```
LOG.error("Failed to recreate node {}", path, e);
```